cmd/ttn-lw-cli/internal/util/flags.go

```diff
@@ -23,6 +23,7 @@ import (
 	"time"
 
 	"github.com/gogo/protobuf/proto"
+	"github.com/gogo/protobuf/types"
```

Comment:
We import this package as `pbtypes` @htdvisser

Reply:
Only if we already import `go.thethings.network/lorawan-stack/pkg/types`